test/extensions/geoip_providers/maxmind/BUILD

```diff
@@ -16,6 +16,7 @@ envoy_extension_cc_test(
     size = "small",
     srcs = select({
         "//bazel:linux": ["config_test.cc"],
+        "//bazel:darwin_any": ["config_test.cc"],
```

Comment:
There is another path for this: https://github.com/envoyproxy/envoy/pull/32489. Fix temporary included here until the patch is merged

Reply:
can rebase now